README.md

```diff
@@ -26,19 +26,25 @@ cd ..
 ```
 
 ```
-cd nodejs;
```

Comment:
Wow--not sure how those got inserted, but thank you for removing them!

Reply:
Sure thing! Perhaps they were leftover from having all of the commands on one line and the semicolons were needed as delimiters. They weren't hurting anything, but I figured I'd remove them to have all of the language examples aligned with not having them.